client-go/cmd/client/main.go

```diff
@@ -11,6 +12,8 @@ import (
 )
 
 func main() {
+	name := flag.String("name", "Foo-Fighter", "Pass arbitrary string to gRPC call request")
+	flag.Parse()
```

Comment:
Nice and straight forward input. Please add instructions to `./client-go/README.md`